genomesubsampler/parseRepophlan.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017--, genome-subsampler development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# parser for RepoPhlAn-downloaded genomes
+#
+
+import click
+import pandas as pd
+
+
+def parse_repophlan(repophlan_wscores_fp):
+    """ Compute basic statistics of RepoPhlAn-downloaded genomes
+
+    Parameters
+    ----------
+    repophlan_wscores_fp: string
+        file path to RepoPhlAn summary table with scores
+
+    Returns
+    -------
+    list of string
+        Human-readable report of basic statistics of genomes
+    """
+    df = pd.read_table(repophlan_wscores_fp, index_col=0, header=0)
+    out = []
```

Comment:
@sjanssen2 I am afraid that I can't agree. `out` is a piece of sequential, multiple-line information. printing `res['Number of RefSeq genomes']` sounds awkward and not precise. At this stage, these numbers are only for user awareness purpose.